config/downloads.js

```diff
@@ -7,12 +7,12 @@ module.exports = [
     color: '#e8433e',
     githubRepo: 'apache/apisix',
     githubBranch: 'master',
-    downloadPath: 'apisix/3.8.0/apache-apisix-3.8.0-src',
+    downloadPath: 'apisix/3.9.0/apache-apisix-3.9.0-src',
     dockerhubPath: 'apisix',
-    version: '3.8.0',
+    version: '3.9.0',
     LTSDownloadPath: 'apisix/3.2.2/apache-apisix-3.2.2-src',
     LTSVersion: '3.2.2',
-    releaseDate: '2024-01-15',
+    releaseDate: '2024-03-25',
```

Comment:
I put March 29 in this PR: https://github.com/apache/apisix-website/pull/1777 (the date of the announcement email)

Reply:
updated